test/test.c

```diff
@@ -21,6 +21,9 @@ static bool setInitialState(app_role_t role,
 {
     WPC_stop_stack();
 
+    // Wait for stack to stop
+    usleep(3 * 1000 * 1000);
```

Comment:
I don't understand why you need it, as WPC_stop_stack already internally wait up to 60sec to get status of stack (60sec being exaggerated, but should cover a very very long time spent in boot-loader to process a scratchpad)

Reply:
Ok, I will investigate it a bit more